src/App.tsx

```diff
@@ -1,46 +1,19 @@
 /* tslint:disable:max-classes-per-file */
 // TODO(zhyty): reenable max-classes-per-file once we're past playing around.
 import * as React from 'react';
+import { Link } from 'react-router-dom';
 
-import * as Snippet from './Snippet';
+import { ExampleNames } from './Examples';
 
-// TODO(zhyty): these are just stubs. Will be need to be updated when Lessons
-// are implemented.
-interface IExercise {
-  title: string;
-  // we'll need to fill this out.
-  content: Snippet.ISnippetProps;
-}
-
-interface ILesson {
```

Comment:
Question: it seems like we're moving away from "Lessons" and focusing more on specific examples? I think this simplifies things for the time being, but just wanted to make sure this is where we're going.

Reply:
That was the idea here, I think we can bring back lessons later if thats where we want to go with it